tests/data/missing_dependency_relationships/SPDXJsonExample.json

```diff
@@ -1,4 +1,10 @@
 {
+    "creationInfo" : {
+        "comment" : "This package has been shipped in source and binary form.\nThe binaries were created with gcc 4.5.1 and expect to link to\ncompatible system run time libraries.",
+        "created" : "2010-01-29T18:30:22Z",
+        "creators" : [ "Tool: LicenseFind-1.0", "Organization: ExampleCodeInspect ()", "Person: Jane Doe ()" ],
+        "licenseListVersion" : "3.17"
+      },
     "Document": {
```

Comment:
This key needs to be deleted as it is no longer valid since 2.2 (see the [schema](https://github.com/spdx/spdx-spec/blob/development/v2.2.2/schemas/spdx-schema.json) also the examples for [SPDX 2.2](https://github.com/spdx/spdx-spec/blob/development/v2.2.2/examples/SPDXJSONExample-v2.2.spdx.json) and [SPDX 2.3](https://github.com/spdx/spdx-spec/blob/development/v2.3.1/examples/SPDXJSONExample-v2.3.spdx.json). I would recommend to replace all test documents with the valid examples from the spec repo and then adjust them according to the test cases.

Reply:
@meretp, good call. I'll do that. Thank you!